planning/backlog.md

```diff
@@ -8,42 +8,38 @@ A collection of user stories you can choose from. Stories inside of each priorit
 
 are necessary for basic usability:
 
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- ...
+- **Website Visitor**: As a Website Visitor I want to have existing header and footer navs so that I have clear navigation around the website:    
+  - [ ] Given when the user visits the website then it must have the main navigation menu displayed in the header and the secondary navigation menu in the footer
+- **Website Visitor**: As a Website Visitor I want the correct links exist in the navs
+  - [ ]  Given when I see the header and the footer, then links to [... specific sections ...] of the website are provided
```

Comment:
Given when I see the header and the footer, then links to Home, Features and Docs pages of the website are provided

Reply:
you are correct !